cobigen-cli/cli/src/test/resources/testdata/openAPI file.yml

```diff
@@ -0,0 +1,111 @@
+openapi: 3.0.0
```

Comment:
Why don't you use the yml file we already have? Please see [here](https://github.com/devonfw/tools-cobigen/blob/master/cobigen-cli/cli/src/test/resources/testdata/openAPI.yml)

Reply:
done